optionals/compat_ffaamod/CfgGlasses.hpp

```diff
@@ -0,0 +1,16 @@
+#define COMBAT_GOGGLES         ACE_Overlay = z\ace\addons\goggles\textures\HUD\CombatGoggles.paa; \
+        ACE_OverlayCracked = z\ace\addons\goggles\textures\HUD\CombatGogglesCracked.paa; \
+        ACE_Resistance = 2; \
+        ACE_Protection = 1;
```

Comment:
This would look better like that:
```
#define COMBAT_GOGGLES \
    ACE_Overlay = z\ace\addons\goggles\textures\HUD\CombatGoggles.paa; \
    ACE_OverlayCracked = z\ace\addons\goggles\textures\HUD\CombatGogglesCracked.paa; \
    ACE_Resistance = 2; \
    ACE_Protection = 1;
```